config/src/main/java/org/springframework/security/config/http/DefaultFilterChainValidator.java

```diff
@@ -53,9 +54,8 @@
 import org.springframework.security.web.servletapi.SecurityContextHolderAwareRequestFilter;
 import org.springframework.security.web.session.SessionManagementFilter;
 import org.springframework.security.web.util.matcher.AnyRequestMatcher;
-import org.springframework.security.web.util.matcher.RequestMatcher;
 
-public class DefaultFilterChainValidator implements FilterChainProxy.FilterChainValidator {
+public final class DefaultFilterChainValidator implements FilterChainProxy.FilterChainValidator {
```

Comment:
This isn't passive, so let's leave it out of the PR. For example, it would break an application that is extending `DefaultFilterChainValidator`.

Reply:
Done